_sass/footer.scss

```diff
@@ -103,10 +102,7 @@ footer {
 
 	a {
 		text-decoration: none;
-
-		&:hover {
-			color: #fff;
-		}
+		color: #fff;
```

Comment:
This change seemed to make the links in the footer not highlight on hover anymore.

Reply:
My issue with the links is that they didn't really stand out. As in you wouldn't know that the "here" was actually a link. Some other links didn't change in any way on hover either. I modified it further in https://github.com/PathOfBuildingCommunity/pathofbuildingcommunity.github.io/pull/19/commits/e235590cfb39d395101e2fbdc4768b3390504c71 to make all links in the body of the page get an underline when hovered over. If they're in the middle of a sentence i made them bold to make them stand out a bit more.

Reply:
This is that link in Firefox and Chrome on prod when hovered:
![image](https://github.com/user-attachments/assets/a4b508c6-23fd-48aa-b8dd-6ce07373f535)

Looked exactly the same in this PR when I tested it yesterday too.  Other links behave the same, other than the ones I mentioned at the bottom.  Is that not the case for you?  Every link does something on hover for me.

Reply:
Ahh. The `See all releases` didn't change so i focused on fixing that and in the process i guess i broke the other ones as well and thought they needed fixing. I'll revert most of the changes. Current prod has them underlined always. Thoughts on making them bold on hover? Or just making them get the underline only on hover like the other ones?

Reply:
Latest commit looks good, thanks for the tweaks!